geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -263,31 +263,22 @@ some `docker build`, and `docker-compose` commands.
 Main Changes
 ------------
 
-Environment variables are now stored in the env.default and env.project files.
-
-The migration process will do the following UI-related changes (visible in the html diffs):
+The migration process will changes the following things:
+1. Environment variables are now stored in the env.default and env.project files.
+2. .mako files are replaced by .tmpl, this means Python templating isn't possible anymore, (in .tmpl we just have template replacement).
+If dynamic variables is needed, you has to write a custom script in `script/config-eval-templates` and add it to the entrypoint into the config image
```

Comment:
"If dynamic variables is needed, you has" -> "If dynamic variables are needed, you have"